packages/message-encryption/src/ecies.ts

```diff
@@ -81,11 +78,24 @@ class Encoder implements IEncoder {
   }
 }
 
-export interface EncoderOptions extends BaseEncoderOptions {
+export interface EncoderOptions {
```

Comment:
why not extend the one from `waku/core`?

Reply:
Because why the point? it just creates indirection for little value.

Reply:
I'll revert this change.